glean-core/src/metrics/ping.rs

```diff
@@ -130,6 +138,10 @@ impl PingType {
         self.0.include_info_sections
     }
 
+    pub(crate) fn schedules_pings(&self) -> &Vec<String> {
```

Comment:
```suggestion
    pub(crate) fn schedules_pings(&self) -> &[String] {
```

it's unusual to return a reference to a vec.